thoth/adviser/run.py

```diff
@@ -110,23 +111,30 @@ def subprocess_run(
                     "automatically marked for solving by the system for future resolutions",
                     ", ".join(exc.unresolved)
                 )
-            _LOGGER.error("Resolver run failed: %s", str(exc))
-            result_dict.update(dict(error=True, error_msg=str(exc), report=exc.to_dict()))
+                return_code = 2
+            else:
+                _LOGGER.error("Resolver run failed: %s", str(exc))
```

Comment:
shall we put the user or ops error tag in the log message too?

Reply:
I don't know what do you mean by that. What would it be used for?

Reply:
@goern ping